docs/contributor-guide/developer-guide/authentication-providers.md

```diff
@@ -8,6 +8,89 @@ This page will go over the implementation and configuration steps for currently
 
 ### Azure AD (OIDC)
 
+Below you'll find instructions on how to use Azure Active Directory to authenticate CiviForm and Program admins. High-level steps are to create an app in Azure AD, create a group that contains CiviForm admins and finally update CiviForm server config to use the app and the group we just created. These instructions are just an example of how to get basic authentication working. Administrators are encouraged to adapt Azure AD part to fit their needs. 
+
+#### Configure Azure AD
+
+1. Login to Azure, go to "Active Active Directory" => "App Registrations" and create a new app. During creation set `Redirect URI` to be https://your-civiform-domain.gov/callback/AdClient replacing domain with your actual domain. The path `/callback/AdClient` is mandatory.
+   <details>
```

Comment:
This is a really neat idea. I love how screenshots are not overwhelming the page, but there for reference.